uber/config.py

```diff
@@ -302,6 +324,10 @@ def _unrepr(d):
 for _opt, _val in chain(_config.items(), c.BADGE_PRICES.items()):
     if not isinstance(_val, dict) and not hasattr(c, _opt.upper()):
         setattr(c, _opt.upper(), _val)
+for _opt, _val in c.BADGE_PRICES['stocks'].items():
+    _opt = _opt.upper() + '_STOCK'
+    if not hasattr(c, _opt):
+        setattr(c, _opt, _val)
```

Comment:
See ``configspec.ini`` below for an explanation, but I added a new ``[[stocks]]`` config section where we can track how many of each badge type we have available.  As per the ``c.*_AVAILABLE`` implementation, it's totally optional, but it allows us to automatically cut off selling badges of each type after a certain number.